manifests/c/Corel/MindManager/23.1.243/Corel.MindManager.installer.yaml

```
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.1.5 $debug=QUSU.CRLF.5-1-19041-1682.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+# Latest download URL: https://www.mindmanager.com/mm-win-enterprise-admin-64
+
+PackageIdentifier: Corel.MindManager
+PackageVersion: 23.1.243
+MinimumOSVersion: 10.0.0.0
+InstallerType: zip
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.DotNet.Framework.DeveloperPack_4
+  - PackageIdentifier: Microsoft.EdgeWebView2Runtime
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+Installers:
+- Architecture: x64
+  NestedInstallerType: msi
+  NestedInstallerFiles:
+  - RelativeFilePath: MindManager 23.msi
+  InstallerUrl: https://download.mindjet.com/MindManager_64bit_23.1.243_Admin.zip
```

```suggestion
  InstallerUrl: https://dwnld.mindmanager.com/stubs/Release/MindManager23/MindManager23_setup.exe
```

Hi @FaserF,

The `PackageUrl` gave me a very different `InstallerUrl`.

- Should this one be used instead?
- Can you show me where the ZIP installer was found?

---

The MSI is the one recommended from the manufacturer for deployments. Therefore I choosed the MSI variant, which also is better for companys deploying the application with winget. 
https://download.mindjet.com/MindManager_Large_Scale_Deployment_Guide_v_15_0.pdf 

This is the always latest link for the MSI variant: https://www.mindmanager.com/mm-win-enterprise-admin-64

Nevertheless, I have now also added the exe parameters. Commit will follow shortly